ee/clickhouse/models/action.py

```diff
@@ -41,7 +41,7 @@ def format_action_filter(
             prop_query, prop_params = parse_prop_clauses(
                 Filter(data={"properties": step.properties}).properties,
                 team_id=action.team.pk if filter_by_team else None,
-                prepend="action_props_{}".format(action.pk),
+                prepend="action_props_{}_{}".format(action.pk, step.pk),
```

Comment:
Would be nice to use [f-strings](https://zetcode.com/python/fstring/) if possible, but not a big deal